README-zh_CN.md

```diff
@@ -288,6 +288,15 @@
 ***
 &nbsp;
 
+## <a id="DarkMode"></a>Dark Mode - Day and nigth
```

Comment:
this section should be part of Chinese doc. 

you can either drop this change or use Google translator to translate the texts